### PR TITLE
updated readme and resource name

### PR DESCRIPTION
**Pull request by ummiyah-moe**

### CODE REVIEW

**Review by sudip-moengage — CHANGES_REQUESTED** [review]

File: autoscaling_group/locals.tf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 locals {
   _subservice_name     = coalesce(var.subservice_name, var.service_name)
   _service_type        = var.service_type != "" ? format("%s-", var.service_type) : ""
-  _resource_identifier = "${local._service_type}${var.business_name}-${var.service_name}"
+  _resource_identifier = "${var.business_name}-${var.service_name}-${var.subservice_name}"
```
[the remark this answers is not in the document]
Comment (ummiyah-moe) [reply autoscaling_group/locals.tf:4]: So if service name not provided, name resources will have double service name. like campaign-flows-flows. or we should mandate SubService name. -  _subservice_name     = coalesce(var.subservice_name, var.service_name)

File: autoscaling_group/README.md [hunk 2]
```diff
@@ -1,55 +1,64 @@
...
+| termination\_policies | A list of policies to decide how the instances in the auto scale group should be terminated. The allowed values are OldestInstance, NewestInstance, OldestLaunchConfiguration, ClosestToNextInstanceHour, OldestLaunchTemplate, AllocationStrategy, Default | `list(string)` | <pre>[<br>  "OldestInstance"<br>]</pre> | no |
+| vpc\_id | VPC ID | `any` | n/a | yes |
+| wait\_for\_capacity\_timeout | A maximum duration that Terraform should wait for ASG instances to be healthy before timing out. Setting this to 0 causes Terraform to skip all Capacity Waiting behavior. Default: 10m | `string` | `"10m"` | no |
+| wait\_for\_elb\_capacity | Setting this will cause Terraform to wait for exactly this number of healthy instances from this autoscaling group in all attached load balancers on both create and update operations. (Takes precedence over min\_elb\_capacity behavior.) | `number` | `0` | no |
 
 ## Outputs
 
 | Name | Description |
 |------|-------------|
-| asg\_instance\_security\_group\_id |  |
```
[the remark this answers is not in the document]
Comment (ummiyah-moe) [reply autoscaling_group/README.md:52]: readme was not  updated in this module so i have run terraform-docs command, didn't update any varibale.

File: autoscaling_group/locals.tf (hunk 1 above)
[the remark this answers is not in the document]
Comment (ummiyah-moe) [reply autoscaling_group/locals.tf]: removed _service_type its not being used anywhere else. and updated with local._subservice_name

File: autoscaling_group/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ummiyah-moe) [reply autoscaling_group/README.md:52]: asg\_instance\_security\_group\_id  does not exist in core module.

**Review by sudip-moengage — APPROVED** [review]

**Pull request merged**